internal/api/api_test.go

```diff
@@ -45,7 +45,10 @@ func setupAPIForTestWithCallback(cb func(*conf.GlobalConfiguration, *storage.Con
 		cb(nil, conn)
 	}
 
-	return NewAPIWithVersion(config, conn, apiTestVersion), config, nil
+	a := NewAPIWithVersion(config, conn, apiTestVersion)
+	a.smsRateLimiter = newUnlimitedLimiter()
+	a.emailRateLimiter = newUnlimitedLimiter()
```

Comment:
does this disable the rate limiting when running the tests?

Reply:
Yes, the tests were failing when the rate limiter was active. Tests that want to make sure limits are imposed will need some additional book keeping. I provided an example of what that can look like inside `ratelimits_test.go`.